scipy/stats/tests/test_stats.py

```diff
@@ -9536,3 +9536,132 @@ def test_chk_asarray(xp):
     x_out, axis_out = _chk_asarray(x[0, 0, 0], axis=axis, xp=xp)
     xp_assert_equal(x_out, xp.asarray(np.atleast_1d(x0[0, 0, 0])))
     assert_equal(axis_out, axis)
+
+
+@make_xp_test_case(stats.wasserstein_distance)
```

Comment:
This is one class for two functions. How do we indicate that with `make_xp_test_case`?